2022-02-23 11:10:27,963 INFO    Thread-18 :1698414 [wandb_setup.py:_flush():75] Loading settings from /home/eunchan/.config/wandb/settings
2022-02-23 11:10:27,963 INFO    Thread-18 :1698414 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'entity': 'eckim', 'project': 'sweeps-random', 'root_dir': '/home/eunchan/Desktop/Study/Wandb-sweeps', 'run_id': '84ojv4ty', 'sweep_param_path': '/home/eunchan/Desktop/Study/Wandb-sweeps/wandb/sweep-233dsju3/config-84ojv4ty.yaml', 'sweep_id': '233dsju3'}
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': 'train.py'}
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_init.py:_log_setup():386] Logging user logs to /home/eunchan/Desktop/Study/Wandb-sweeps/wandb/run-20220223_111027-84ojv4ty/logs/debug.log
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_init.py:_log_setup():387] Logging internal logs to /home/eunchan/Desktop/Study/Wandb-sweeps/wandb/run-20220223_111027-84ojv4ty/logs/debug-internal.log
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_init.py:init():420] calling init triggers
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_init.py:init():425] wandb.init called with sweep_config: {'batch_size': 89, 'dropout': 0.5, 'epochs': 6, 'learning_rate': 0.03634331967746129, 'optimizer': 'sgd'}
config: {'epochs': 2, 'batch_size': 128, 'fc_layer_size': 128, 'weight_decay': 0.0005, 'learning_rate': 0.001, 'activation': 'relu', 'optimizer': 'adam', 'seed': 42}
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [wandb_init.py:init():471] starting backend
2022-02-23 11:10:27,964 INFO    Thread-18 :1698414 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 11:10:27,966 INFO    Thread-18 :1698414 [backend.py:ensure_launched():219] starting backend process...
2022-02-23 11:10:27,994 INFO    Thread-18 :1698414 [backend.py:ensure_launched():224] started backend process with pid: 1698918
2022-02-23 11:10:27,995 INFO    Thread-18 :1698414 [wandb_init.py:init():480] backend started and connected
2022-02-23 11:10:27,995 INFO    Thread-18 :1698414 [wandb_run.py:_config_callback():966] config_cb None None {'batch_size': 89, 'dropout': 0.5, 'epochs': 6, 'learning_rate': 0.03634331967746129, 'optimizer': 'sgd'}
2022-02-23 11:10:27,997 INFO    Thread-18 :1698414 [wandb_init.py:init():550] updated telemetry
2022-02-23 11:10:27,998 INFO    Thread-18 :1698414 [wandb_init.py:init():581] communicating current version
2022-02-23 11:10:28,408 INFO    Thread-18 :1698414 [wandb_init.py:init():586] got version response 
2022-02-23 11:10:28,408 INFO    Thread-18 :1698414 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-23 11:10:28,678 INFO    Thread-18 :1698414 [wandb_init.py:init():624] starting run threads in backend
2022-02-23 11:10:30,014 INFO    Thread-18 :1698414 [wandb_run.py:_console_start():1827] atexit reg
2022-02-23 11:10:30,014 INFO    Thread-18 :1698414 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-02-23 11:10:30,014 INFO    Thread-18 :1698414 [wandb_run.py:_redirect():1706] Redirecting console.
2022-02-23 11:10:30,015 ERROR   Thread-18 :1698414 [wandb_run.py:_redirect():1765] Failed to redirect.
Traceback (most recent call last):
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1758, in _redirect
    out_redir.install()
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-02-23 11:10:30,015 INFO    Thread-18 :1698414 [wandb_init.py:init():651] run started, returning control to user process
2022-02-23 11:10:30,064 INFO    Thread-18 :1698414 [wandb_watch.py:watch():43] Watching
2022-02-23 11:10:36,644 INFO    Thread-18 :1698414 [wandb_run.py:finish():1468] finishing run eckim/sweeps-random/84ojv4ty
2022-02-23 11:10:36,644 INFO    Thread-18 :1698414 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-23 11:10:36,645 INFO    Thread-18 :1698414 [wandb_run.py:_restore():1769] restore
2022-02-23 11:10:36,646 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 535
}

2022-02-23 11:10:36,921 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 535
}

2022-02-23 11:10:37,695 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 10315
}

2022-02-23 11:10:37,797 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 11006
}

2022-02-23 11:10:37,897 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 11006
}

2022-02-23 11:10:38,000 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4116
  total_bytes: 11006
}

2022-02-23 11:10:38,103 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4116
  total_bytes: 11006
}

2022-02-23 11:10:38,207 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:38,310 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:38,413 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:38,517 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:38,618 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:38,890 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}

2022-02-23 11:10:39,388 INFO    Thread-18 :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11006
  total_bytes: 11006
}
local_info {
}

2022-02-23 11:10:40,572 INFO    Thread-18 :1698414 [wandb_run.py:_append_history():2144] rendering history
2022-02-23 11:10:40,573 INFO    Thread-18 :1698414 [wandb_run.py:_append_summary():2102] rendering summary
2022-02-23 11:10:40,573 INFO    Thread-18 :1698414 [wandb_run.py:_append_files():2194] logging synced files
